CHANGES.txt

```diff
@@ -13,7 +13,8 @@ https://mhammond.github.io/pywin32_installers.html .
 
 Coming in build 310, as yet unreleased
 --------------------------------------
-* Drop support for Vista, set Windows 7 as the minimal Windows version. (#, @Avasam)
+* Fixed a regression where `win32com.client.DispatchWithEvents` and win32com.client.WithEvents` would throw a `TypeError` on the second call (#2491, @Avasam)
+* Drop support for Vista, set Windows 7 as the minimal Windows version. (#2487, @Avasam)
   * Restores many IIDs in `win32com(ext).shell.shell`. See #2486 for details.
```

Comment:
The indentation of this line looks wrong now there are other entries - should it be dedented?

Reply:
I meant for it to be a precision. But I could rewrite as "2 entries, added by the same changeset" like this: 
```suggestion
* Restored many IIDs in `win32com(ext).shell.shell`. See #2486 for details. (#2487, @Avasam)
```

Reply:
That "sub item" is more important imo - so maybe it should also move up and note it's a regression?  code using win32com.shell is likely broken, right?

Reply:
Feel free to reorganize the changelog to your likings if you want keep more important stuff at the top. You have the final say on those before releases ^^